tests/update/patch_before_migration.pm

```diff
@@ -0,0 +1,76 @@
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Patch SLE qcow2 images before migration (offline)
+# Maintainer: Dumitru Gutu <dgutu@suse.de>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+use registration;
+
+sub patching_sle() {
+
+    select_console 'root-console';
+
+    set_var("VIDEOMODE",    'text');
+    set_var("SCC_REGISTER", 'installation');
+    set_var("SP2ORLATER",   '0') if (!sle_version_at_least('12-SP2', version_variable => 'HDDVERSION'));
```

Comment:
@mitiao, I've encounter a problem to register scc addons in text mode after your changes here - https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2138 
Without this I have troubles for sles12ga and sles12sp1.